portfolio/src/main/webapp/comments.js

```diff
@@ -93,7 +96,22 @@ function createCommentElement(text) {
   return commentElement;
 }
 
-/** Posts a comment to the server */
+/** 
+ * Create the container for the timestamp of individual posts
+ * @param {int} postTime in milliseconds
+ * @return {HTMLElement} the html element containing the @param postTime
+ */
+function createPostTimeElement(postTime) {
+  let postTimeElement = document.createElement('small');
+  let postTimeString = '\n' + new Date(postTime).toLocaleString();
```

Comment:
Comment explaining why '\n' is needed here would be good

Reply:
```suggestion
  // Insert a newline character to separate the comment from its timestamp
  let postTimeString = '\n' + new Date(postTime).toLocaleString();
```